microsoft-edge/visual-studio/index.md

```diff
@@ -95,18 +92,18 @@ The example was just a minor demonstration of the functionality available in Vis
 
 ## Attach to Microsoft Edge  
 
-Previously, you had to launch Microsoft Edge from Visual Studio.  Now, you are able to attach the Visual Studio debugger to an already running instance of Microsoft Edge.  
+Previously, you launched Microsoft Edge from Visual Studio.  You can also attach the Visual Studio debugger to an already running instance of Microsoft Edge.  
 
 First, ensure that there are no running instances of Microsoft Edge.  Now, from your command line, run the following command.  
 
 ```console
 start msedge –remote-debugging-port=9222
 ```  
 
-From Visual Studio, open the **Debug** menu and choose **Attach to Process** or select `Ctrl`+`Alt`+`P`.  
+From Visual Studio, select **Debug** > **Attach to Process** or `Ctrl`+`Alt`+`P`.  
 
-:::image type="complex" source="./media/attach-to-process.png" alt-text="Choose Attach to Process in Visual Studio" lightbox="./media/attach-to-process.png":::
-   Choose **Attach to Process** in Visual Studio  
+:::image type="complex" source="./media/attach-to-process.png" alt-text="Select Attach to Process in Visual Studio" lightbox="./media/attach-to-process.png":::
+   Select **Attach to Process** in Visual Studio  
 :::image-end:::  
 
 From the **Attach to Process** dialog, set **Connection type** to **Chrome devtools protocol websocket (no authentication)**.  In the **Connecting target** textbox, type `http://localhost:9222/` and select `Enter`.  Review the list of open tabs you have in Microsoft Edge listed out in the **Attach to Process** dialog.  
```

Comment:
```suggestion
1.  In the **Attach to Process** dialog, set **Connection type** to **Chrome devtools protocol websocket (no authentication)**.  
1.  In the **Connecting target** textbox, type `http://localhost:9222/` and select `Enter`.  In the **Attach to Process** dialog, review the list of tabs that are open in Microsoft Edge.  
```
Break into 2-3 cohesive steps.  Reading this step out of context, it's unclear how many dialog boxes are involved.  Each dialog would be a separate step.